images/win/scripts/Tests/Tools.Tests.ps1

```diff
@@ -91,6 +91,12 @@ Describe "Mingw64" {
     }
 }
 
+Describe "Microsoft GDK" {
```

Comment:
You should add a condition to skip this test for Windows Server 2016/2022

Reply:
Updated